contentcuration/contentcuration/views/settings.py

```diff
@@ -65,7 +65,7 @@ def settings(request):
 @login_required
 @api_view(['GET'])
 def export_user_data(request):
-    generateusercsv_task.delay(request.user.pk)
+    generateusercsv_task.delay(request.user.pk, language=request.LANGUAGE_CODE)
```

Comment:
I think it is marginally safer to use this utility function instead of the request LANGUAGE_CODE:
https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.translation.get_language

Same below.